.eslintrc.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
     'no-alert': 'error',
     // Stylistic opinion
     'arrow-body-style': 'off',
+    'import/no-named-as-default': 'off',
```

Comment:
This shouldn't be necessary after updating the imports as per my comment in packages/mui-base/src/useBadge/index.ts.